src/for_mathlib/sheaves/presheaf_of_rings.lean

```diff
@@ -48,6 +48,6 @@ structure iso (F G : presheaf_of_rings α) :=
 (mor_inv_id : mor.to_morphism ⊚ inv.to_morphism = presheaf.id F.to_presheaf)
 (inv_mor_id : inv.to_morphism ⊚ mor.to_morphism = presheaf.id G.to_presheaf)
 
-infix `≅`:80 := λ A B, nonempty (iso A B)
+local infix `≅`:80 := λ A B, nonempty (iso A B)
```

Comment:
What is the point of defining a *local* notation at the end of file? It clearly shows it won't be used anywhere

Reply:
Fixed